Disaster-Response-Platform/backend/Models/feedback_model.py

```diff
@@ -0,0 +1,20 @@
+from pydantic import BaseModel, Field
+from typing import Dict, Any
+from enum import Enum
+
+
+class Feedback(BaseModel):
+    _id: str = Field(default=None)
+    entityType: str = Field(default=None)
+    entityID: str = Field(default=None)
+    username: str = Field(default=None)
+    vote: str = Field(default=None)
+    
+    
+    
+class VoteUpdate(BaseModel):
+    entityType: str
```

Comment:
I think it will be better to provide an enumration for entityType.

Reply:
thanks for the suggestion, I'll do it with enum